pkg/plugins/hybrid/v1alpha/scaffolds/init.go

```diff
@@ -120,21 +119,30 @@ func (s *initScaffolder) Scaffold() error {
 
 	err = scaffold.Execute(
 		&templates.Main{},
-		&templates.GoMod{ControllerRuntimeVersion: golangv4.ControllerRuntimeVersion},
+		// TODO: This used to be golangv4.ControllerRuntimeVersion to be in sync
+		// with the version scaffolded by the version of Kubebuilder we depend on.
+		// As of writing this comment, Kubebuilder is behind on the kube bumps
+		// so we are manually setting this to the needed controller-runtime version
+		// to facilitate the bump to support k8s 1.30
```

Comment:
Seems like since we own `main.go` for the hybrid plugin, we should also just own the `go.mod`.

So unless I'm missing something, I don't think we ever need to switch back to golangv4's controller-runtime version?

Reply:
Ah, @varshaprasad96 commented elsewhere that we do actually use golangv4 in other places for the hybrid plugin. So I can see how this would be unsafe (e.g. if golangv4 `create api` is used, it probably doesn't work with controller-runtime 0.18.x).

Reply:
Addressed @varshaprasad96 's comment in https://github.com/operator-framework/helm-operator-plugins/pull/337/commits/70c5aaaf1998f3504ca8f257b21d0327382f8b3f